docs/process.md

```diff
@@ -2127,6 +2127,34 @@ Resource labels are currently only supported by the {ref}`awsbatch-executor`, {r
 
 See also: [label](#label)
 
+(process-resourcelimits)=
+
+### resourceLimits
+
+:::{versionadded} 23.08.0-edge
+:::
+
+The `resourceLimits` directive allows you to specify environment-specific limits for task resource requests. Resource limits can be specified in a process as follows:
+
+```groovy
+process my_task {
+    resourceLimits [
```

Comment:
```suggestion
    resourceLimits = [
```

Reply:
The equals sign is used in the config but not in the process definition. I have added examples for both.

Reply:
Oh I see, it's clear now, thanks!